modules/statistics/statistics.js

```diff
@@ -270,8 +270,11 @@ Statistics.prototype.startCallStats = function (session, settings) {
 Statistics.prototype.stopCallStats = function () {
     if(this.callstats) {
         var index = Statistics.callsStatsInstances.indexOf(this.callstats);
-        Statistics.callsStatsInstances.splice(index, 1);
-        this.callstats = null;
+        if(index > -1)
+            Statistics.callsStatsInstances.splice(index, 1);
+        // The next line is commented because we need to be able to send
```

Comment:
I would rather see new method JitsiConference.dispose() added to JitsiConference and make  .leave() keep some parts of the conference alive(including Call stats) than doing this kind of hacks. But it's only my personal opinion.